api/server/server.go

```diff
@@ -177,7 +177,7 @@ func (s *Server) makeHTTPHandler(handler httputils.APIFunc) http.HandlerFunc {
 // InitRouters initializes a list of routers for the server.
 func (s *Server) InitRouters(d *daemon.Daemon) {
```

Comment:
This init function should take a backend. I'm fine if it is done in a follow on PR.